ocs_ci/ocs/constants.py

```diff
@@ -1067,6 +1130,7 @@
 VSPHERE_PLATFORM = "vsphere"
 BAREMETAL_PLATFORM = "baremetal"
 IBM_POWER_PLATFORM = "powervs"
+IBM_CLOUD_BAREMETAL_PLATFORM = "ibm_cloud_baremetal"
```

Comment:
I'm thinking if we have to separate the `baremetal` and `ibm_cloud_baremetal` platforms?

Reply:
rename ibm_cloud_baremetal -> ibm_baremetal ?

Reply:
No I meant if we could use the existing `BAREMETAL_PLATFORM`? Or if there are some incompatible differences?

Reply:
I think there maybe differences between baremetal and IBM baremetal

Reply:
The question will be, how big the differences will be? For example in my PR https://github.com/red-hat-storage/ocs-ci/pull/9451, I'm handling it as generic baremetal platform and the difference between the BM env in Octo lab  and the IBM Cloud BM is in the [`mgmt_provider`](https://github.com/red-hat-storage/ocs-ci/pull/9451/files#diff-05053122d6e5936ca528fcd1643379f792a8350f094e1c0aa5c62903f86a8746R1184), which controls if the BM machine is managed via `ipmitool` or `ibmcloud` command.

I don't know all the details there about the differences, so maybe it make sense to separate it like this - but if the differences are not too big, I would vote to keep it unified here.

Reply:
for node operations the differences are serious @yitzhak12 can confirm